packages/ui/components/ui/tx/actions-views/swap/one-way-swap.tsx

```diff
@@ -1,4 +1,4 @@
-import { ValueViewComponent } from '../value';
+import { ValueViewComponent } from '../../../value';
 import { ArrowRight } from 'lucide-react';
 import { ValueView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/asset/v1/asset_pb';
 import { getAmount, getSymbolFromValueView } from '@penumbra-zone/getters/value-view';
```

Comment:
on line 21, instead of `{outputAmount ? <ValueViewComponent view={output} /> : getSymbolFromValueView(output)}` should be `<ValueViewComponent view={output} showValue={!!outputAmount} />`